framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -490,6 +490,14 @@ int cfg_get_hostap_iface(int32_t radio_num, char hostap_iface[BPL_IFNAME_LEN]);
  */
 int cfg_get_all_prplmesh_wifi_interfaces(BPL_WLAN_IFACE *interfaces, int *num_of_interfaces);
 
+/**
+ * @brief Get persistent-db enable.
+ * 
+ * @param enable [in/out] Enable persistent DB read from config - default value (if parameter is not configured) is DEFAULT_PERSISTENT_DB.
```

Comment:
According to Doxygen it should be `[in,out]` and not '[in/out]', and if I'm not mistaken it should be after the `@param`.
Second, where DEFAULT_PERSISTENT_DB is configured? I don't see it on the master, this and the previous commits, which means that this commit breaks the build, and it shouldn't.

Also `DEFAULT_PERSISTENT_DB` seems to me that it could be used internally only on the cpp file, and should not be exposed on bpl_cfg.h. 
In fact, I don't think that it is even a good idea to change it when the function fails to get the parameter.
Let the user decide what to do in case of failure.
So I would remove the whole " - default value ..." part, and remove DEFAULT_PERSISTENT_DB definition as well.

Reply:
I'll start with the end:
the suggestion that if persistent-db doesn't exist in the configuration should lead to "persistent-db=disable" was raised by @arnout and I agree with him (this is important for transparency to a platform that does not want to implement the persistent-db. it should specify literally that it is false, instead - by not stating it is enable (or not being aware this feature was added at all) the feature is not enabled for that system.

regarding the DEFAULT_PERSISTENT_DB - I fixed it and it is now introduced in the right commit